grblas/matrix.py

```diff
@@ -561,62 +556,67 @@ def _prep_for_assign(self, resolved_indexes, obj):
                     raise TypeError(f'Expected Matrix for assignment value; found {type(obj)}')
                 delayed = GbDelayed(lib.GrB_Matrix_assign,
                                     [obj.gb_obj[0], rows, rowsize, cols, colsize],
-                                    at=obj.is_transposed)
-
+                                    at=obj._is_transposed)
```

Comment:
`isinstance(obj, Matrix)` check above needs to also check for TransposedMatrix

Reply:
Good catch!  Fixed and added a test to exercise this.